CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Exceptions are acceptable depending on the circumstances (critical bug fixes tha
 - changed the OSD version to 2.16.0 due to an upgrade request
 - **BREAKING CHANGE**: changed `global/scripts/golang/test/run.sh` to execute `go test` for all folders instad for `main` or `cmd`
 - changed `run.sh` `goland` test script to install docker used by test containers in integration tests
+- changed `run.sh` `goland` test script to run integration tests separately and once per time
```

Comment:
The name of the language should be "GoLang" right? GoLand is the IDE from JetBrains.